server/src/seeders/20210111120513-demo-data-set.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+const v4 = require('uuid').v4;
+const bcrypt =  require('bcryptjs');
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        const date = new Date();
+        const org_id_svm = v4();
+        const user_id_svm_admin = v4();
+        const team_id_svm_outdoor = v4();
+        const role_id_svm_admin = v4();
+        const organizations = [{
+            id: org_id_svm,
+            access_code: 'svm2020',
+            config: '{}',
+            title: 'SVM',
+            is_active: true,
+            admin_role_id: role_id_svm_admin,
+            createdAt: date,
+            updatedAt: date
+        }];
+        queryInterface.bulkInsert('Organizations', organizations, {});
+        
+        const roles = [{
+            id: role_id_svm_admin,
+            title: 'SVM Admin',
```

Comment:
```suggestion
            title: 'Alle Administratoren',
```

Reply:
Besser: Wir legen in der Organization eine Hook an, welche das direkt nach `AfterCreate` anlegt. Geht fürs erste aber auch so.

https://github.com/RobinBuschmann/sequelize-typescript/blob/master/test/models/Hook.ts#L14

https://github.com/RobinBuschmann/sequelize-typescript#hooks